docs/user/dashboard/lens.asciidoc

```diff
@@ -96,13 +96,37 @@ All columns that belong to the same layer pane group are sorted in the table.
 
 * *Text alignment* &mdash; Aligns the values in the cell to the *Left*, *Center*, or *Right*.
 
+* *Color by value* &mdash; Applies color to the cell or text values. To change the color, click the *Edit colors* icon. 
+
 * *Hide column* &mdash; Hides the column for the field.
 
 * *Directly filter on click* &mdash; Turns column values into clickable links that allow you to filter or drill down into the data.
 
 * *Summary row* &mdash; Adds a row that displays the summary value. When specified, allows you to enter a *Summary label*.
 
-* *Color by value* &mdash; Applies color to the cell or text values. To change the color, click *Edit*. 
+[float]
+[[assign-colors-to-terms]]
+===== Assign colors to terms
+
+preview::[]
+
+Assign a color to each term in a table with color mapping.
+
+. Create a custom table.
+ 
+. In the layer pane, select a *Rows* or *Metrics* field.
```

Comment:
It's kind of implied in the section title but might be worth clarifying this is a selection of a term based metric and not a numeric one. If they were to click on a numeric one they will not see the new color mapping feature.